tests/openssl_sgx_tcrypto_compat/host/CSchema_checker.cpp

```diff
@@ -0,0 +1,358 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <new>
+
+#include "CSchema_checker.h"
+#include "include_openssl.h"
+
+CSchemaChecker::CSchemaChecker(t_openssl_schema* schema, uint schema_size)
+{
+    m_schema_idx = 0;
+    m_schema_num_apis = schema_size;
```

Comment:
```suggestion
    m_schema_api_count = schema_size;
```
https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md says:

> DON'T use abbreviations unless they are already well-known terms known by users (e.g., "app", "info"), or are already required for use by developers (e.g., "min", "max", "args"). Examples of bad use would be num_widgets instead of widget_count, and opt_widgets instead of option_widgets or optional_widgets.

Reply:
Updated